addons/mrp/models/product.py

```diff
@@ -46,6 +46,16 @@ def action_view_mos(self):
         }
         return action
 
+    def action_view_bom(self):
+        action = self.env.ref('mrp.template_open_bom').read()[0]
```

Comment:
Alternatively:
```suggestion
        action = self.env['ir.actions.act_window'].fox_xml_id('mrp', 'template_open_bom')
```

Reply:
fox? Seems typo here ( or on the main function)

Reply:
All for_xml_id does is calling ref :smile: